src/compas/data/data.py

```diff
@@ -28,15 +28,77 @@
 class Data(object):
     """Abstract base class for all COMPAS data objects.
 
+    Parameters
+    ----------
+    name : str, optional
+        The name of the object.
+
     Attributes
     ----------
-    DATASCHEMA : :class:`schema.Schema`
-        The schema of the data dict.
-    JSONSCHEMA : dict
-        The schema of the serialized data dict.
+    dtype : str, read-only
+        The type of the object in the form of a fully qualified module name and a class name, separated by a forward slash ("/").
+        For example: ``"compas.datastructures/Mesh"``.
```

Comment:
This is *not* a suggestion for a change in this PR at all, but maybe to start a conversation later on, but, what's the reason for using slash as separator instead of doing a really fully-qualified module & name with dots, ie. `compas.datastructures.Mesh`? Is it some legacy due to the way xfunc used to work?

Reply:
think i got this from pickle and/or numpy